toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/ConfigPanel.java

```diff
@@ -569,58 +570,89 @@ public static boolean testWebAppURL(String URL, boolean runTryValue) throws IOEx
 	public static boolean getStatus(String url) throws IOException {
 
         boolean result = false;
-        try {
-            URL siteURL = new URL(url);
-            HttpURLConnection connection = (HttpURLConnection) siteURL.openConnection();
-            connection.setRequestMethod("GET");
-            connection.connect();
-            int code = connection.getResponseCode();
-            if (code == 200) {
-                result = true;
-            }
-        } catch(SSLHandshakeException sslException){
+		InetSocketAddress proxyInet = new InetSocketAddress("10.23.252.215",3128);
```

Comment:
Yes, but this will be changed once @NathCoco merge the ui modification with the proxy config feature